community/libtpm2-tss/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor:
+# Maintainer:
+pkgname="libtpm2-tss"
+pkgver="2.3.1"
+pkgrel=0
+pkgdesc="TPM2 tss library"
+url="https://github.com/tpm2-software/tpm2-tss/"
+arch="all"
+license="BSD-2-Clause"
+depends=""
+makedepends="automake autoconf autoconf-archive libtool openssl-dev doxygen linux-headers"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/tpm2-software/tpm2-tss/tarball/2.3.1"
+builddir="$srcdir/"tpm2-software-tpm2-tss-"*"
```

Comment:
what's the deal with using a wildcard ?

Reply:
Thanks for your review and teaching a noob the way it goes with alpine. All comments make sense and I will adjust ... but on this one, I might need some guidance:
 
the tarball i get from github has a commit hash at the end (e.g. tpm2-software-tpm2-tss-7f93b34 for release 2.3.1), couldn' find a way to convince abuild to rename the top dir to something that is stable and doesnt' need adjustment on each new release we pull in... 

Let me know!

Reply:
Can't these be used ?

https://github.com/tpm2-software/tpm2-tss/releases/download/2.3.1/tpm2-tss-2.3.1.tar.gz